scripts/test_updates_pg12.sh

```diff
@@ -20,3 +20,5 @@ run_tests "$@" -v7 \
           2.0.0-rc2-pg12 2.0.0-rc3-pg12 2.0.0-rc4-pg12 2.0.0-pg12 2.0.1-pg12 2.0.2-pg12 2.1.0-pg12 \
           2.1.1-pg12 2.2.0-pg12 2.2.1-pg12 2.3.0-pg12 2.3.1-pg12 2.4.0-pg12 2.4.1-pg12 2.4.2-pg12 \
           2.5.0-pg12 2.5.1-pg12
```

Comment:
```suggestion
          2.1.1-pg12 2.2.0-pg12 2.2.1-pg12 2.3.0-pg12 2.3.1-pg12 2.4.0-pg12 2.4.1-pg12 2.4.2-pg12
```
v8 should be a superset of v7 and if it works starting at 2.5.0 then  2.5+ should be removed from v7 here.